src/pages/Provider/Provider.scss

```diff
@@ -1,111 +1,127 @@
 .providerHeader {
-  position: relative;
+  position: sticky;
+  top: 0px;
+  z-index: 100;
+  background-color: $secondary;
+  padding: 1.5625rem 2.3125rem 1.125rem;
+  border-radius: 0px 0px 5px 5px;
 
   &__img {
     max-height: 30vh;
-    width: 100%;
-    object-fit: cover;
-    object-position: 50% 50%;
+    width: 4.375rem;
+    height: 1.875rem;
     aspect-ratio: 16/9;
-    position: relative;
 
-    &::after {
-      content: '';
-      color: red;
-      width: 100%;
+    img {
       height: 100%;
-      display: block;
-      position: absolute;
-      top: 0px;
-      left: 0px;
-      right: 0px;
-      bottom: 0px;
-      background-color: rgba(0, 0, 0, 0.45);
+      width: 100%;
+      object-fit: contain !important;
     }
   }
 
-  &__name {
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translateX(-50%) translateY(-50%);
-    font-size: 4rem;
-    color: white;
+  &__btnBadge {
+    bottom: 0px;
+    top: unset;
   }
 
-  &__backBtn {
-    position: absolute;
-    top: 0px;
-    left: 0px;
-    z-index: 1;
-    width: 40px;
-    height: 40px;
+  &__topContainer {
     display: flex;
-    justify-content: center;
-    align-items: center;
-    color: white;
-    cursor: pointer;
+    justify-content: space-between;
+    margin: 1rem 0px;
   }
 
-  &__tabs {
-    .q-tabs__content {
-      .q-tab {
-        color: darkgray;
-
-        &--active {
-          color: black;
-        }
-
-        .q-tab__indicator {
-          height: 4px;
-          border-radius: 4px;
-        }
-
-        &:first-of-type {
-          justify-content: start;
-          padding-left: 10%;
-
-          .q-tab__indicator {
-            width: 80%;
-          }
-        }
-        &:last-of-type {
-          justify-content: end;
-          padding-right: 10%;
-
-          .q-tab__indicator {
-            width: 80%;
-            margin-left: auto;
-          }
-        }
+  &__form {
+    display: flex;
+    justify-content: space-between;
+    gap: 0.8125rem;
+
+    &_search {
+      width: 100%;
+
+      .q-field__control {
+        border-radius: 0.375rem;
+        height: 3.125rem;
+      }
+
+      .q-field__label {
+        font-weight: 400;
+        font-size: 0.875rem;
       }
     }
+
+    &_button {
+      border-radius: 0.3125rem;
+    }
   }
 }
 
 .Provider {
-  background: rgba($dark, 0.05);
+  background: $white-flight;
   min-height: 100vh;
+
+  &__categories {
+    margin: 1rem 0;
+  }
 }
 
 .ProviderProducts {
   display: grid;
   grid-auto-flow: row;
-  grid-auto-rows: 250px;
-  gap: 40px;
+  grid-auto-rows: 8rem;
+  gap: 1rem;
+  margin: 8px 24px 8px 18px;
```

Comment:
Sometimes rem is used, what about standardize what to use in each case?

Reply:
I think it's ok to use rem for all else by `box-shadows`, what do you think about it?

Reply:
I've made the change to rem for the moment, and I've opened this discussion so we can talk about it https://github.com/Azordev/did-app/discussions/37